README.md

```diff
@@ -15,4 +15,6 @@ These images are based on [this parent image](https://hub.docker.com/r/coqorg/ba
 
 See also the [docker-coq wiki](https://github.com/coq-community/docker-coq/wiki) for details about how to use these images.
 
+This repository is [mirrored on Gitlab](https://gitlab.com/coq-community/docker-coq), but [https://github.com/coq-community/docker-coq/issues] and [pull requests](https://github.com/coq-community/docker-coq/pulls) are tracked on GitHub.
```

Comment:
Thanks @Blaisorblade ! this is definitely a specificity of docker-base and docker-coq (unlike new docker-keeper based projects that will probably be only on GitLab)

can you just fix the 2nd URL:

```suggestion
This repository is [mirrored on Gitlab](https://gitlab.com/coq-community/docker-coq), but [issues](https://github.com/coq-community/docker-coq/issues) and [pull requests](https://github.com/coq-community/docker-coq/pulls) are tracked on GitHub.
```

Reply:
Of course, that’s the danger of web edits and tiredness 😅